scripts/generate-changelog.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Updates Unrealeased section of CHANGELOG.md by generating content with all commited changelog entry files defined after last release.
```

Comment:
@EspenAlbert to the pull request probably. Instead of `issues`, `pull`

Reply:
Correct the intention is to reference the pull request. In github the `issues` can be used to reference a pull request (example for this PR `https://github.com/mongodb/terraform-provider-mongodbatlas/issues/2121`), however will adjust to avoid any confusion.